mod/groups/classes/Elgg/Groups/Group.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Elgg\Groups;
+
+/**
+ * Handle group related hooks and events
+ *
+ * @since 4.0
+ * @internal
+ */
+class Group {
+
+	/**
+	 * Populates the ->getUrl() method for group objects
+	 *
+	 * @param \Elgg\Hook $hook 'entity:url', 'group'
+	 *
+	 * @return void|string
+	 */
+	public static function getEntityUrl(\Elgg\Hook $hook) {
```

Comment:
why? this is handled in the `view:group:group` route

Reply:
https://github.com/Elgg/Elgg/issues/13092